js/openPicture.js

```diff
@@ -0,0 +1,56 @@
+const bigPicture = document.querySelector('.big-picture');
+const body = document.querySelector('body');
+const comment = document.querySelector('.social__comment');
+const commentsList = document.querySelector('.social__comments');
+commentsList.textContent = '';
+const closeButton = document.querySelector('.big-picture__cancel');
+
+const commentCount = document.querySelector('.social__comment-count');
+const loader = document.querySelector('.comments-loader');
+
+const closeBigPicture = () => {
+  body.addEventListener('keydown', (evt) => {
+    if (evt.keyCode === 27) {
+      bigPicture.classList.add('hidden');
+    }
+  });
+
+  closeButton.addEventListener('click', () => {
+    bigPicture.classList.add('hidden');
+  });
+};
+
+
+const openBigPicture = (url, description, comments, likes) => {
+  closeBigPicture();
+  bigPicture.classList.remove('hidden');
+  body.classList.add('modal-open');
+  commentCount.classList.add('hidden');
+  loader.classList.add('hidden');
+
+  const countLikes = bigPicture.querySelector('.likes-count');
+  const photo = bigPicture.querySelector('.big-picture__img').querySelector('img');
+  const photoTitle = bigPicture.querySelector('.social__caption');
+  const commentsCount = bigPicture.querySelector('.comments-count');
+
+  photo.src = url;
+  photo.alt = description;
+  countLikes.textContent = likes;
+  photoTitle.textContent = description;
+  commentsCount.textContent = comments.length;
+
+  comments.forEach((obj) => {
```

Comment:
Указывай сущность с которой работаешь чтобы лишний раз не нагружать того кто смотрит код
```suggestion
  comments.forEach((comment) => {
```